### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by RosaRyni** [yuliskov/SmartTubeNext#729]
Like the other day, the app loads the interface, but refuses to play anything.
Running on MiBox S on 2 different TVs. Was working fine on our living room TV, but not on our bedroom TV. Tried to restart the device, as well as force quitting and clearing data, but no dice.
Restarted living room device and now have the same problem there as well.

**Comment by Venoryk** [comment]
Same, running fire stick 2 and between watching 1 video to selecting another, playback ceased.

**Comment by Bobby1000** [comment]
Same, running on TV. Was watching a video. When it was finished, the next video failed to start. I found I was unable to watch any video.

**Comment by Wolf68k** [comment]
Same here. It was working just fine. I watched a video. Left to room to do something. Came back and nothing.
I didn't restart my TV but from the looks of other reports that wouldn't do much good.

**Comment by frandr0id** [comment]
Stopped working. Black screen on every video I try to play.

**Comment by Djtiestoja** [comment]
Not working here as well. Black screen, refuse to play ▶️

**Comment by HoellenBewohner** [comment]
stopped just working on firetv and android tv
no video is loading > looks like youtube will try to stop adblocking

**Comment by borislip** [comment]
Same here, black screen on every video i try to play, tested on Sony Bravia 2015 and CCwGTV.

**Comment by rainydaydriver** [comment]
Doesnt work anymore, tried both latest release and beta.  Black screen, pressing play doesn't do anything. 0.00/0.00 showed in the video time.  Android TV (hisense H9G).

**Comment by Wolf68k** [comment]
> 
> 
> looks like youtube will try to stop adblocking

I don't think that is it. I think it's more like the History when it suddenly stops working.

**Comment by gotee12** [comment]
Same here. Using a Nvidia Shield 2019.

**Comment by willcaddy** [comment]
I seem to have the same issue on my Sony Android TV. Interface works fine but refuses to play videos.

**Comment by finnzz** [comment]
Update v11.47 has a fix.

**Comment by zymbiosis** [comment]
Confirmed it works, your work is much appreciated!  Donation incoming!

**Comment by gotee12** [comment]
Update resolved it for me as well.

**Comment by gotee12** [comment]
Update resolved it for me as well

**Comment by Bobby1000** [comment]
Fixed. Works very well. Thank you.
